user.html

```diff
@@ -5,14 +5,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="https://fonts.googleapis.com/css?family=Fredericka+the+Great&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="stylesheets/user.css">
     <title>New Tab</title>
 </head>
 <body>
     <img src="images/favicon-128x128.png" class="logo">
     <h1>What's your name?</h1>
     <form action="index.html">
-        <input type="text" name="username" id="usr-input">
+        <input type="text" name="username" id="usr-input" placeholder="Type Here..."> 
```

Comment:
Again, this is not directly related to the issue. A PR should only do what it claims to do. For any additional changes, you are free to add a new issue and send a specific PR related to that. This change is fine, however and you may keep this if you want.

Reply:
Okay, Sorry for that..

Reply:
No problem :smile: